tangelo/linq/translator/translate_json_ionq.py

```diff
@@ -55,22 +65,23 @@ def translate_json_ionq(source_circuit):
     json_gates = []
     for gate in source_circuit._gates:
         if gate.name in {"H", "X", "Y", "Z", "S", "T"}:
-            json_gates.append({'gate': GATE_JSON_IONQ[gate.name], 'target': gate.target[0]})
+            json_gates.append({'gate': GATE_JSON_IONQ[gate.name], 'targets': gate.target})
```

Comment:
I'm a bit confused here. I have a feeling we are only dealing with 4 different situations:

- A list of gates that take no control, no parameters (SWAP, X, Y, Z, H, S, T)
- A list of gates that take parameters but no controls (RX, RY, RZ, PHASE, XX)
- A list of gates that take controls but no parameters (CX, CY, CZ, CNOT)
- A list of gates that take both (CRX, CRY, CRZ, CPHASE)

Couldn't we refactor this to only have 4 branches, now that controls and targets keywords are used everywhere ?